### PR TITLE
created new pom class

### PR DESCRIPTION
**Pull request by samgithubsam100**
please review

### CODE REVIEW

File: Vtiger.com/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.comcast.crm.objectrepositoryutility;
+
+public class CreateProductPage {
+
```
Comment (Sauravgarg1820) [inline Vtiger.com/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java:4]: Change the x-path ,
 don't hardcode the data

**Review by Sauravgarg1820 — COMMENTED** [review]
added few comments

**Comment by samgithubsam100** [comment]
fixed

**Comment by Sauravgarg1820** [comment]
merged

**Pull request merged**